### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by chibomorto** [WeeJeWel/wg-easy#57]
First of all, thanks for the extraordinary work! I know this topic was already covered. But since I'm a noob, can you please clarify if I need to install the Wireguard packages for it to work?
Thanks

**Comment by jieuryli** [comment]
You do not need to install wg-easy on the host machine for this to work. You just need to have Docker installed. All the WG stuff is installed, configured, and runs within the Docker container.

**Comment by Minvinea** [comment]
@chibomorto it's good ?

**Comment by chibomorto** [comment]
> Hi, you want ton install wg-easy on what os ?

I'm using Ubuntu Server 20.04. I have docker installed and i'm running a docker stack with some apps installed with docker-compose.
Do i need to  install Wireguard packages on the host system? 
Thank You

**Comment by Minvinea** [comment]
Yes, it's best install wireguard and after follow the step to config and start your container

**Comment by cb3inco** [comment]
I don't think you do. The dockerfile builds this container with Wireguard already installed.

**Comment by gaby** [comment]
> I don't think you do. The dockerfile builds this container with Wireguard already installed.

As noted in the README: 

> A host with a kernel that supports WireGuard (all modern kernels).

The reason the container has wireguard-tools installed is to facilitate access to `wg-quick` and `wg` cli tools without having to mount the binaries into the container.
